tests/src/utility/testUtility.ts

```diff
@@ -0,0 +1,52 @@
+/**********************************************************************
```

Comment:
filename is odd

we are in tests folder, and utility folder, and name of the file is testUtility ?

I would expect it describes the name of what functions are doing in the file

I would not mix in the same file, functions to remove folder/cleanup that are part of the execution of the tests

and functions that are more 'page objects'

so probably cleanup.ts with removeFolderIfExists

and welcomePage.ts with `handleWelcomePage` function but probably not in 'utility' folder but more in 'pages' folder

Reply:
ok. I wanted to keep that for totally different pr but here we go.